docs/intro/ff-ui/canvas.md

```diff
@@ -0,0 +1,135 @@
+---
+slug: canvas
+title: Canvas
+tags: []
+---
+
+# Canvas 
+The Canvas shows the mobile device screen where you can add widgets by dragging and dropping them. You can select, move, and position widgets anywhere on the Canvas. 
+
+It allows zoom level adjustments and previews in light or dark mode. It also includes features for multi-language preview, adding AppBar and NavBar, simulating larger font visualization, and more.
+
+![CanvasArea.avif](imgs/CanvasArea.avif)
+
+## 1. Preview Screen
+
+This is the canvas of the device screen where you can build the user interface. You can customize the screen by adding widgets using drag and drop from the [Widget Palette](#) and by applying properties present in the [Properties Panel](#).
+
+## 2. Show or hide Navigation Menu
+
+From here, you can open or close the [Navigation Menu](#).
+
+## 3. Zoom Controls
+
+There's a zoom in ( + ) and a zoom out ( - ) button to control the zoom level of the canvas. While working on complex user interface designs, this comes in handy to give you a better preview of the widgets by zooming in to a particular area and then zooming out to have an overview.
+
+## 4. AI Generated Page
+
+From here, you can [create a page using AI](#).
+
+## 5. Add AppBar
+
+From here, you can add an [AppBar](#) to your page. Clicking this button opens a popup displaying different [AppBar styles](#) for you to choose from. Just select any AppBar style from the list and it will appear in the Preview Screen.
+
+![AddAppBar.avif](imgs/AddAppBar.avif)
+
+## 6. Multi-Language Preview
+
+If you've enabled the multi-language feature for your project, you can use this to preview your app in different languages. Just select the language from the dropdown menu, and the text in your app will be translated accordingly.
+
+:::info
+This becomes really helpful in testing your app design with translated text without having you run your app.
+:::
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'
+}}>
+    <iframe 
+        src="https://demo.arcade.software/ViB374dOSglo4Z1afa0o?embed&show_copy_link=true"
```

Comment:
This is a lot of zoom, specially in terms of zoom size. It's pixelating as you zoom. I think a small video clip in between the slides as you change the language will give an idea to the users that text changed. 

And If we want to zoom here, we can do a minimal zoom size change and stay there throughout without moving too much.